sources/packages/backend/apps/db-migrations/src/sql/ApplicationRestrictionBypasses/Create-application-restriction-bypasses.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE sims.application_restriction_bypasses (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  student_restriction_id INT NOT NULL REFERENCES sims.student_restrictions(id),
+  bypass_behavior sims.restriction_bypass_behaviors NOT NULL,
+  is_active BOOLEAN NOT NULL,
+  creation_note_id INT NOT NULL REFERENCES sims.notes(id),
+  bypass_created_by INT NOT NULL REFERENCES sims.users(id),
+  bypass_created_date TIMESTAMP WITH TIME ZONE NOT NULL,
+  removal_note_id INT REFERENCES sims.notes(id),
+  bypass_removed_by INT REFERENCES sims.users(id),
+  bypass_removed_date TIMESTAMP WITH TIME ZONE,
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id),
+  modifier INT NULL DEFAULT NULL REFERENCES sims.users(id)
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.application_restriction_bypasses IS 'Restrictions bypass that allow awards to be disbursed ignoring restrictions at the student application level.';
+
+COMMENT ON COLUMN sims.application_restriction_bypasses.id IS 'Auto-generated sequential primary key column.';
+
+COMMENT ON COLUMN sims.application_restriction_bypasses.application_id IS 'Reference to the student application that will have the bypass applied.';
+
+COMMENT ON COLUMN sims.application_restriction_bypasses.student_restriction_id IS 'Active student restriction to be bypassed.';
```

Comment:
The reason for this column is that we can have specific restrictions that are bypassed and not by passed or the is there a group of restrictions that cannot be bypassed itself? Because having this isactive in here will be for a particular application. Please confirm my understanding.

Reply:
I am not sure if I followed the question.
One Student restriction can be bypassed for 1 to n applications. These bypasses can be added and removed, setting `is_active` to false.

All restrictions in the student account can be bypassed, and the bypasses should be individually added to each application as required. Please let's have a call if it is not clear.